### ISSUE TITLE
Multi threading version

### ISSUE THREAD

**Issue by trueman1990** [ROSS-org/ROSS#156]
Is still an Multi-threading version offer now?

**Comment by laprej** [comment]
tl;dr No, ROSS does not currently support threading.

As far as I know all threading support within ROSS has been removed.  It happened quite some time ago when the decision was made to only support MPI.  In recent years, however, some of the supercomputer vendors are once again encouraging the use of threads.  @carothersc may have an old version of ROSS that still has threading support.

**Comment by caitlinross** [comment]
Yeah, Justin is right. I removed any remaining threading support recently because there are no plans to go to a hybrid MPI/pthreads version of ROSS. There is a shared memory version which still uses only MPI, but MPI ranks that are colocated on a compute node use event buffers from a shared memory segment for on-node events.  This hasn't been merged into the master branch of ROSS yet though for various reasons.

**Comment by suchenjm** [comment]
Is there a Multi-threading version recommended here?
